networkx/readwrite/graphml.py

```diff
@@ -81,6 +82,8 @@ def write_graphml_xml(
        we infer in GraphML that both are floats.
     named_key_ids : bool (optional)
        If True use attr.name as value for key elements' id attribute.
+    edge_id_from_attribute : keyword argument, hashtable identifier (optional),
+       Select edge_attribute for edge_id
```

Comment:
I think this parameter description needs to be made more clear, e.g.

```suggestion
    edge_id_from_attribute : dict key (optional),
       If provided, the graphml edge id is set by looking up the corresponding
       edge data attribute keyed by this parameter. If `None`, the edge id is set
       by the edge key if `G` is a MultiGraph, else the edge id is left unset.
```

It's difficult to capture the behavior in a concise description, but I think something along these lines would be a bit more clear. It'd be great to include an example in the docstring, but it's not necessary for this PR.

Reply:
Thanks for the review. I add one extra sentance to make the description of default situation clearer.
I will provide some examples in the docstring after the PR is merged.
``` python
"""
edge_id_from_attribute : dict key (optional)
    If provided, the graphml edge id is set by looking up the corresponding
    edge data attribute keyed by this parameter. If `None` or the key does not exist in edge data, 
    the edge id is set by the edge key if `G` is a MultiGraph, else the edge id is left unset.
"""
```